### ISSUE TITLE
The system log displays a `qca-nss-ecm` problem！

### ISSUE THREAD

**Issue by zxlhhyccc** [qosmio/nss-packages#19]
![image](https://github.com/qosmio/nss-packages/assets/45259624/8b46640b-71c1-46cb-83ac-beae7428469b)

**This is caused by the following code, and this definition is not found in the script:**

[qosmio/nss-packages/blob/NSS-12.4-K6.1/qca-nss-ecm/files/qca-nss-ecm.init#L77](https://github.com/qosmio/nss-packages/blob/NSS-12.4-K6.1/qca-nss-ecm/files/qca-nss-ecm.init#L77)

**Comment by qosmio** [comment]
Looks like you're using a custom `ecm` config? Try the following.

```uci
➤ cat /etc/config/ecm
config ecm 'global'
        option acceleration_engine 'nss'

config general
        option enable_bridge_filtering '0'
        option disable_offloads '0'
        option disable_flow_control '0'
        option disable_interrupt_moderation '0'
        option disable_gro '0'
```

**Comment by zxlhhyccc** [comment]
> Looks like you're using a custom `ecm` config? Try the following.
> 
> ```
> ➤ cat /etc/config/ecm
> config ecm 'global'
>         option acceleration_engine 'nss'
> 
> config general
>         option enable_bridge_filtering '0'
>         option disable_offloads '0'
>         option disable_flow_control '0'
>         option disable_interrupt_moderation '0'
>         option disable_gro '0'
> ```

![image](https://github.com/qosmio/nss-packages/assets/45259624/72a65232-64ad-4fb1-a798-a5e6c1855ab6)
![image](https://github.com/qosmio/nss-packages/assets/45259624/1cef3015-f567-42ae-b176-aefcc72187e0)

`ecm`config add ` option disable_gro '0'` normal。How is  ` option disable_gro '0'` generated? This item was not present in the original configuration！

**Comment by qosmio** [comment]
fixed in 486a2c1f34a7807eb249a97c83e1a63424e623e9
